### ISSUE TITLE
Delay initialization after other service providers?

### ISSUE THREAD

**Issue by itsgoingd** [itsgoingd/clockwork#464]
Eg. in combination with `vyuldashev/laravel-queue-rabbitmq` we can currently crash as we try to initialize queue before the other package had a chance to add the rabbitmq driver.

**Comment by nathan-io** [comment]
After installing Clockwork in a Laravel 8 application which uses `vyuldashev/laravel-queue-rabbitmq`, I'm getting "InvalidArgumentException No connector for [rabbitmq]" in `vendor/laravel/framework/src/Illuminate/Queue/QueueManager.php:172`.

**Comment by itsgoingd** [comment]
Here's a workaround:

Disable auto-loading of Clockwork in your `composer.json`:

```
    "extra": {
        "laravel": {
            "dont-discover": [
                "itsgoingd/clockwork"
            ]
        }
    },
```

Now register the service provider manually in `config/app.php`:

```php
    'providers' => [
        ...
        Clockwork\Support\Laravel\ClockworkServiceProvider::class,
        ...
    ],
```

This will cause Clockwork to be initializer *after* `vyuldashev/laravel-queue-rabbitmq`, which solves the issue.

**Comment by nathan-io** [comment]
Thanks, that solved it!

**Comment by itsgoingd** [comment]
Fix included in Clockwork 5.0.7.
